content/docs/user-guide/networking/multiplayer/overview.md

```diff
@@ -45,6 +45,30 @@ Then, clients update themselves from a delta by:
 
 After, the AzNetworking layer becomes aware that the world was updated. This occurs automatically through ACK vector replication, and does not require any further action from the client.
 
+### Types of network properties
+
+Network properties have two important fields: `ReplicateFrom` and `ReplicateTo`. 
+Together, these define which role can replicate to which role. You can only replicate property values *from* Authority and Autonomous roles. Under the server-authority model of O3DE, this ensures that you never accidentally replicate from an unprivileged client. 
+Properties can be replicated *to* any role, since all active clients in the session must share information and some may not necessarily need to be used in the server-authoritative model.
+
+| Field | Description | Values | 
+| - | - | - |
+| `ReplicateFrom` | Tells the network which role can send information about the changes that were made to this property. | Authority, Autonomous | 
+| `ReplicateTo` | Tells the network which role can receive information about the changes that were made to this property. | Server, Authority, Autonomous, Client | 
+
+
+The following are four types of networking properties with a valid "from and to" relationship that you can define, and their possible use cases:
+
+- **Authority-to-Client**: For handling client, or "simulated", properties.
+
+- **Authority-to-Autonomous**: For handling autonomous-only properties.
+
+- **Authority-to-Server**: For handling host migration.
+
+- **Autonomous-to-Authority**: For gathering information about client metrics, such as monitoring the health of clients.
+
+For networking properties that replicate from Authority, a replication hierarchy applies. The replicate-to rules trickle up the hierarchy in the following way: An Authority-to-Client replication also replicates to Autonomous and Server roles. An Authority-to-Autonomous replication also replicates to the Server role. Finally, an Authority-to-Server replication only replicates to the Server. This behavioral hierarchy ensures that if the Authority ever migrates to the other server, then the other server has the right property information.
+
 ## Remote procedure calls (RPCs)
 
 Remote procedure calls are how O3DE allows for pushing messages between hosts, rather than sending an update that's based on the server world state. O3DE offers both *reliable* and *unreliable* RPCs. By default, RPCs are reliable.
```

Comment:
@lmbr-pip  I'd certainly like to improve this - do you estimate that its a quick fix (I can do in this PR) or shall I create an issue for us to improve this at a separate time?

Reply:
Suggestion:
_Remote procedure calls are how O3DE allows for pushing messages between multiplayer endpoints, rather than sending an update that's based on the server world state (ie: network properties). O3DE offers both *reliable* and *unreliable* RPCs. By default, RPCs are reliable. Note: RPCs aren’t guaranteed to arrive in the order they are sent._

Reply:
Echo @AMZN-puvvadar with the addition that _RPCs aren’t guaranteed to arrive in the order they are sent._

Reply:
Thank you all for the explanation - I'll get this in